doc/lightning-listpeerchannels.7.md

```diff
@@ -194,4 +194,7 @@ Main web site: <https://github.com/ElementsProject/lightning> Lightning
 RFC site (BOLT \#9):
 <https://github.com/lightningnetwork/lightning-rfc/blob/master/09-features.md>
 
-[comment]: # ( SHA256STAMP:269c1f688cb6b3d42da9899386eabc48691a78987cfb455fafc4c2656df2b7df)
+<<<<<<< HEAD
+=======
+>>>>>>> 9d1d65a1b (msggen: Regenerate for ListPeerChannels)
+[comment]: # ( SHA256STAMP:c8cc5cce6119eeec5551d513f7cf60fdd21842f0f446263a37cce3fa3d0c1e33)
```

Comment:
Oops, you committed a conflict marker!

Hmm, perhaps we should propagate "added" flags downwards in msggen?  By adding it to every field, it's almost useless for the documentation.  Though in this case, adding it to the schema fields is probably overkill anyway?